### PR TITLE
[91] - switching the patterns in the MediaInternetGovCmsCkanHandler->…

### PR DESCRIPTION
**Pull request by kurtfoster**
https://github.com/govCMS/govcms-ckan/issues/91

### CODE REVIEW

**Comment by kurtfoster** [comment]
https://github.com/govCMS/govcms-ckan/issues/91

**Comment by seanhamlin** [comment]
Requested reviews from some of the primary contributors.

File: modules/govcms_ckan_media/includes/MediaInternetGovCmsCkanHandler.inc [hunk 1]
```diff
@@ -34,8 +34,8 @@ class MediaInternetGovCmsCkanHandler extends MediaInternetBaseHandler {
     // @baseurl/dataset/PACKAGE_ID/resource/RESOURCE_ID/download/filename.csv
     // @baseurl/dataset/PACKAGE_ID/resource/RESOURCE_ID
     $patterns = array(
-      '@' . $this->baseHost . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)/download/(?:[^"\&\? ]+)@i',
-      '@' . $this->baseHost . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)@i',
+      '@' . $this->baseUrl . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)/download/(?:[^"\&\? ]+)@i',
+      '@' . $this->baseUrl . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)@i',
```
Comment (jeremy-doghouse) [inline modules/govcms_ckan_media/includes/MediaInternetGovCmsCkanHandler.inc]: That one was changed to support ckan endpoints using different ports, could we instead change where baseUrl is set to also include the path if it exists? 

Eg. https://github.com/govCMS/govcms-ckan/blob/7.x-1.x/modules/govcms_ckan_media/includes/MediaInternetGovCmsCkanHandler.inc#L182

**Comment by kurtfoster** [comment]
@seanhamlin , @jeremy-doghouse , I've made a small update to include the baseHost and a basePath.

**Comment by jeremy-doghouse** [comment]
Approved pending changing the base branch to develop. @srowlands or @steveworley can you please have a look at this too.

**Review by steveworley — APPROVED** [review]

**Comment by kurtfoster** [comment]
@tobybellwood @steveworley @srowlands , any idea on a timeline for getting this into govCMS?

**Review by stooit — APPROVED** [review]

**Pull request merged**